cross/create-ndk-standalone.sh

```diff
@@ -37,3 +26,4 @@ echo 'Updating .cargo/config.toml...'
 cd ..
 mkdir -p .cargo
 sed 's|$PWD|'"${PWD}"'|g' cargo-config.toml.template > .cargo/config
+mv .cargo ..
```

Comment:
Actually, I think it's better if we don't move it :) That way we can keep the link configs local to this directory. This means though that you have to build from that directory.